src/bundle/Resources/views/content/tabs/relations.html.twig

```diff
@@ -0,0 +1,30 @@
+<table class="ez-table-data">
```

Comment:
add this to the top of the file {% trans_default_domain "locationview" %} then the 2nd param for trans can be omitted.